.github/workflows/ci.yml

```diff
@@ -26,7 +26,22 @@ jobs:
 
       - name: Lint
         run: npm run lint
+  markdown:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
 
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Lint
+        run: npm run markdownlint
```

Comment:
Could we add this to the `lint` job as a step after `npm run lint`? Would be great to keep it in the same job for linting, something like:

```
      - name: Lint JavaScript
        run: npm run lint
      - name: Lint markdown
        run: npm run markdownlint
```

Reply:
Sure, I can do that! I got [feedback](https://github.com/primer/design/pull/345#discussion_r1055753994) in the `primer/design` repo that it's more clear when there are separate linting jobs for error clarity, but we can keep it together if you prefer. I don't have a strong opinion.

Reply:
Updated!